dubbo-registry/dubbo-registry-nacos/src/main/java/org/apache/dubbo/registry/nacos/util/NacosNamingServiceUtils.java

```diff
@@ -133,7 +133,7 @@ private static void setServerAddr(URL url, Properties properties) {
     }
 
     private static void setProperties(URL url, Properties properties) {
-        putPropertyIfAbsent(url, properties, NACOS_NAMING_LOG_NAME);
+        putPropertyIfAbsent(url, properties, NACOS_NAMING_LOG_NAME, null);
```

Comment:
`null` should not be default value here. We should not call `properties.setProperty` with null.

Reply:
Also refer to Integration Tests